package.json

```diff
@@ -5,8 +5,15 @@
   "version": "2.1.0",
   "author": "Evert Timberg <evert.timberg@gmail.com>",
   "license": "MIT",
-  "main": "dist/chartjs-plugin-annotation.js",
-  "module": "dist/chartjs-plugin-annotation.esm.js",
+  "type": "module",
+  "main": "dist/chartjs-plugin-annotation.esm.js",
```

Comment:
If we want to support chart.js v3 on node/commonjs with these later annotation plugin versions, this needs to be umd build with .cjs extension

Something like I did in autocolors: https://github.com/kurkle/chartjs-plugin-autocolors/blob/main/package.json

Reply:
Ok Let me update in next hours

Reply:
@kurkle updated as suggested